src/DDTrace/Integrations/Mongo/MongoIntegration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace DDTrace\Integrations\Mongo;
+
+use DDTrace\Span;
+use DDTrace\Tags;
+use DDTrace\Types;
+use DDTrace\Integrations\Integration;
+
+final class MongoIntegration
+{
+    public static function load()
+    {
+        if (!extension_loaded('mongo')) {
+            return;
+        }
+        MongoClientIntegration::load();
```

Comment:
We should add this at least to Laravel4 integration. We can hold on the other ones until we have a way to disable specific integrations.